tests/hook_implementations/test_hook_implementations.py

```diff
@@ -12,7 +12,7 @@
 
 
 def test_hook_implementations():
-    pm = ErtPluginManager(
+    pluginmanager = ErtPluginManager(
```

Comment:
plugin_manager?

Reply:
Fixed. `plugin_manager` is in line with the spelling inside the module `ert.shared.plugins`. Hopefully that will not lead to more confusion.